app/scripts/components/common/card.tsx

```diff
@@ -214,18 +272,19 @@ const CardFigure = styled(Figure)`
 `;
 
 interface CardComponentProps {
-  title: string;
+  title: React.ReactNode;
```

Comment:
Just out of curiosity: do we have a set practice on using React types directly, rather than importing them (`import React, { ReactNode } from 'react';`)

Reply:
I don't think we do. Do you favor one?

Reply:
@nerik Did this in a separate PR because there were a lot of changes. https://github.com/NASA-IMPACT/veda-ui/pull/517